test/features/steps/smoke_meetings.py

```diff
@@ -23,3 +23,42 @@ def step_impl(context):
 @then(u'I expect a response status code \'{status_code}\'')
 def step_impl(context, status_code):
     expect(str(context.response.status_code)) == status_code
+
+
+########################
+
+
+@given(u'I make a \'{method}\' request to \'{endpoint}\' by Id')
```

Comment:
this step already exists in the file common steps
"@step(u'I set {method} to {endpoint}')"